docs/scripts/gen-faq.js

```diff
@@ -0,0 +1,151 @@
+// read.js
+const fs = require('fs')
+const yaml = require('js-yaml')
+const { Remarkable } = require('remarkable')
+const path = require('path')
+const yamlPath = path.resolve('./docs/faq.yaml')
+
+// Generating FAQ.mdx
+
+if (!fs.existsSync(yamlPath)) {
+  //file exists
+  console.warn('.yaml File does not exists, skipping generating FAQ')
+  return 0
+}
+
+let faqYaml = fs.readFileSync(yamlPath, 'utf8')
+let faq = yaml.load(faqYaml)
+
+const tags = Array.from(new Set(faq.map(({ tags }) => tags).flat(1)))
+
+let data = `---
+id: faq
+title: Frequently Asked Questions (FAQ)
+---
+
+
+
+import {Question, Faq} from '@theme/Faq'
+
+<Faq tags="${tags.join(' ')}"/>
+<br/><br/>
+
+`
+const md = new Remarkable()
```

Comment:
Aren't you rendering markdown in the first place? Why do you need this?

Reply:
In fact, we're rendering html embedded in a mdx-file. Here is an example:
```
<Question tags="question_src-theme- kratos_src-theme- database_src-theme-">
#kratos #database <br/>
<p><strong>Q</strong>: We want to seperate our customers and employees, so we store them in different databases. But we would like to have them use the same login dialog for our portal. How can I achieve that with Ory Kratos?</p>
<p><strong>A</strong>: You can deploy Ory Kratos two times, and use the same login/registration endpoints if you like. You may need to tell you login/registration ui which Kratos it is supposed to talk to.</p>
</Question>
```
We do that as we need to insert the classes into the questions and we created an additional Docusaurus-plugin (in Faq.js) in order to achieve that. 
Makes sense?!